ecpy_pulses/infos.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015 by Ecpy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Objects used to store filters, sequences and configs in the manager.
+
+"""
+from __future__ import (division, unicode_literals, print_function,
+                        absolute_import)
+
+
+from atom.api import Atom, Subclass, Dict, Coerced
+import enaml
+
+from .sequences.base_sequences import AbstractSequence
+from .configs.base_config import AbstractConfig
+from .contexts.base_context import BaseContext
+from .shapes.base_shapes import AbstractShape
+from .pulse import Pulse
+
+with enaml.imports():
+    from .sequences.views.abstract_sequence_view import AbstractSequenceView
+    from .configs.base_config_views import AbstractConfigView
+    from .shapes.views.base_shapes_views import AbstractShapeView
+    from .contexts.views.base_context_view import BaseContextView
+    from .pulse_view import PulseView
+
+
+class ObjectDependentInfos(Atom):
```

Comment:
This is not needed. It exists for tasks because of instrument dependencies, only context may have the same kind of behavior. If the need appear we can always add support using metadata.